### ISSUE TITLE
Skin Server Switch

### ISSUE THREAD

**Issue by DarianLStephens** [elifoster/WitcheryPatch#7]
Until recently, Mojang had been maintaining both the old skin servers and the new one, and Witchery had been pulling from the old one.
Now, that server has been shut down, so Witchery can't find your skin. That means things like sleeping players, reflections, Hobgoblin statues, and anything else pointing to that server, now show a default Steve skin.
I would really like it if you would make the mod instead point towards the new skin server, to allow it to continue working properly.

TL;DR: Witchery used old skin servers that were recently shut down, so all skin-related things are Steve.
Please make the mod point to the new skin server.

**Comment by xJon** [comment]
Use LumySkinPatch
https://minecraft.curseforge.com/projects/lumy-skin-patch/

**Comment by DarianLStephens** [comment]
Oh, I forgot that I had an issue here.
Yeah, I discovered that ages ago, and it's been working flawlessly.
